src/styles/pages/home.css

```diff
@@ -181,3 +181,233 @@
   opacity: 1;
   visibility: visible;
 }
+
+.transaction-list-container {
```

Comment:
Seem like you put all your style into home.scss. So what 7-1 pattern css for?

Reply:
I fixed it @phuocduong-agilityio  . Please help me check. Thanks !